submissions/chabuChabu/css/styles.css

```diff
@@ -0,0 +1,466 @@
+	display: block;
+	width: 90%;
+	border-radius: 5px;
+	border:1px solid #e0e0e0;
+	outline: none;
+	color: #444;
+	line-height: 1.5rem;
+	margin: 0 auto;
+
+}
+
+#form-label{
+padding-left: 25px;
+}	
+
+a{
+	text-decoration: none;
+}
+
+header{
+	background-color: #69629f;
+	height: 66px;
+	
+}
+
+header *{
+	color: white;
+}
+
+header .logo{
+	float: left;
+	height: inherit;
+	margin-left: 2em;
+	
+}
+
+header .logo-text{
+	margin: 9px;
+}
+
+header ul {
+	float: right;
+	margin: 0px
+	padding:0px;
+	list-style: none;
+	
+}
+
+header ul li{
+	float: left;
+	
+}
+
+header ul li a{
+	display: block;
+	padding: 21px;
+	text-decoration: none;
+
+}
+
+header ul li a:hover {
+	background-color: #5e578e;
+	transition: 0.5s;
+}
+
+.page-wrapper{
+	min-height: 100%;
+	
+}
+
+/**--Content--**/
+.content{
+	width: 90%;
+	margin: 10px auto 30px;
+}
+
+.content .main-content{
+	width: 58%;
+	float: left;	
+}
+
+.content .main-content .registration-title{
+   
+   font-size: 48px; 
+   color: #69629f;
+   padding-top: 140px;
+   width: 50%;
+   margin: 0 auto;
+  
+}
+
+.content .main-content .section-one{
+	width: 50%;
+	margin: 0 auto;
+	border-radius: 5px;
+	font-size: 25px;
+	color: grey;
+}
+
+.content .main-content .section-one .reg-wording{
+  width: 130%;
+
+}
+
+.content .main-content .section-one .reg-wording a{
+  font-weight: bold;
+
+}
+
+/**Sidebar styles**/
+
+.content .sidebar {
+	width: 30%;
+	float: left;
+	border: 1px solid grey;
+	border-radius: 10px;
+	
+}
+
+.content .sidebar .section {
+	padding: 20px;
+	border-radius: 5px;
+    margin-bottom: 20px;
+}	
+
+.btn{
+	width: 50%;
+	margin: 0 auto;
+	background-color: #69629f;
+	color: #fff;
+	font-weight: bold;
+	padding: .7rem 1rem;
+	border-radius: 5px;
+	display: block;
+	margin: auto;
+	border: none;
+	margin-bottom: 10px;	
+}
+
+#terms{
+	color: #69629f;
+	display: block;
+	padding-top: 15px;
+	text-align: center;
+}
+
+/**Footer**/
+
+.footer{
+	background-color: #69629f;
+	/**padding: 4rem;**/
+	padding: 10px;
+}
+.footer-content{
+	width: 100%;
+	height: 150px;
+	background-color: #69629f;
+	color: white;
+	display: flex;
+	justify-content:space-between;
+	flex-wrap: wrap;
+	margin: 0 auto;
+}
+
+.footer .footer-section a{
+	color: white;
+	text-decoration: none;
+	list-style-type: none;
+
+}
+
+.footer .footer-section ul{
+	list-style-type: none;
+}
+
+.footer .footer-content .footer-section input[type="emailaddress"]{
+    width: 170px;
+	border-radius: 5px;
+	padding: 8px;
+	border: none;
+
+}
+.footer .footer-content .footer-section input[type="button"]{
+    background-color: white;
+	color: #69629f;
+	font-weight: bold;
+    width: 100px;
+	padding: 5px;
+	height: 32px;
+	border: none;
+}
+.footer .footer-content .footer-section input[type="image"]{
+  border-radius: 11px;
+  margin: 8px;
+
+}
+.footer-copyright{
+  background:#69629f;
+  color: white;
+  text-align: center;
+}
+
+
+
+/** end of footer**/
+::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
```

Comment:
@ChabuChabu You can group line 226 -237

Reply:
Done

Reply:
This is not how to group selectors. Check the slides for grouping selectors. @ChabuChabu